style.css

```diff
@@ -372,3 +371,199 @@ fieldset > * {
   align-self: center;
   border-radius: 10px;
 }
+
+/* Desktop version using media query */
+@media only screen and (width >= 768px) {
+  .nav-container {
+    display: flex;
+    padding: 16px 144px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-container ul {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+  }
+
+  .skills-languages li {
+    display: flex;
+    flex-direction: column;
+    padding: 12px;
+  }
+
+  .nav-container ul li {
+    display: flex;
+    margin-right: 20px;
+  }
+
+  .main {
+    background-image: url(images/desktopbg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    padding: 289px 260px;
+    border-bottom-left-radius: 80px;
+  }
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    flex-shrink: 0;
+  }
+
+  .main h1 {
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .main p {
+    color: #344563;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .main h3 {
+    color: #7f8cff;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 12px;
+  }
+
+  /* work section started for desktop */
+  .work-section {
+    display: grid;
+    padding: 142px 142px;
+    gap: 142px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    gap: 24px;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card-text {
+    display: flex;
+    flex-direction: column;
+    padding: 24px;
+    max-width: 100%;
+    width: 50%;
+  }
+
+  .card img {
+    max-width: 100%;
+    width: 50%;
+  }
+
+  .card-text h2 {
+    font-size: 40px;
+    font-weight: 700px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .card-text p {
+    font-size: 18px;
+    line-height: 24px;
+    font-weight: 400;
+    color: #6b778c;
+  }
+
+  .card-text strong {
+    color: #344563;
+    font-weight: 500;
+  }
+
+  .card-text i {
+    color: #c1c7d0;
+    margin: 0 4px;
+  }
+
+  .card-text p.detail {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  /* Start of about section for desktop */
+  .aboutpage {
+    display: flex;
+    flex-direction: row;
+    padding: 142px;
+    border-top-right-radius: 120px;
+    justify-content: flex-start;
+    gap: 24px;
+  }
+
+  .top {
+    flex-basis: 41%;
+  }
+
+  .top h2 {
+    color: #344563;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .top p {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .bottom {
+    flex-basis: 51%;
+  }
+
+  .skills-languages {
+    display: flex;
+    flex-direction: row;
+    gap: 12px;
+  }
```

Comment:
Nice job making your page look responsive, you did a good job👍. But currently, your skills section looks a bit different from the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=FmIFt6ll0n8P7fuA-0)😉. Kindly take note of the spacings around the language images😉.

Nice job sticking to the design for the popup on the mobile screen😉 but we are also [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) to stick to the design even as we make the popup responsive😉👍 Let's make some adjustments to our popup on the desktop screen so it can look more like the given design👍👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="381" alt="image" src="https://github.com/anshuman7negi/my-portfolio/assets/64643995/00cacef5-7049-4fee-9e4d-fef1fb23a4d5">|<img width="437" alt="image" src="https://github.com/anshuman7negi/my-portfolio/assets/64643995/3c598f9c-f8c1-4467-8fc3-6a317015c2a6">|

also take note of the font-weight of the texts and make sure they match the given design😉.